Templates/Snackbar/snackbar.css

```diff
@@ -0,0 +1,38 @@
+.snack {
+  background-color: var(--dark-gray);
+  max-width: 50rem;
+  color: var(--snow);
+  border-radius: 5px;
+  box-shadow: 6px 6px 9px var(--gray);
+}
+.normalSnack {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
You can make this a utility class and use that

Reply:
yes actually, I thought of using lesser utility classes to keep the html cleaner. The justify and align properties are something which I hardly use in my current layout(I use space around sometimes, center other times, etc.), so I thought of avoiding making it into a utility class. If I do encounter more such cases, I will make utility class for it. Thanks for pointing it out.

Reply:
Even I'm using fewer utility classes for the same reason but yeah if you don't have a separate utility class it's fine